content/articles/how-to-create-nlp-application-with-flair/index.md

```diff
@@ -0,0 +1,424 @@
+Flair is a natural language processing (NLP) used to build machine learning models used in text classification and speech recognition. It is used in language translation applications, speech recognition, and conversion of speech to text and text to speech applications. It's made up of a text embedding library that enables one to combine different word and document embeddings during model building.
+
```

Comment:
Flair is a natural language processing used....

This sentence is not correct. Please correct it.